run.sh

```diff
@@ -0,0 +1,40 @@
+
+if [[ "$1" == "antwerp" ]]; then
+
+    module use /apps/antwerpen/modules/centos8/software-admin-x86_64
+    module load ReFrame/3.9.1
+
+    export RFM_CONFIG_FILE=$PWD/config_vsc.py
+    reframe --verbose --checkpath $PWD/tests --recursive --run
+
+        
+elif [[ "$1" == "ghent" ]]; then
+
+    # specify Slurm account to use (credits)
+    export SBATCH_ACCOUNT='gadminforever'
+
+    module load ReFrame/3.9.1
+    cd vsc-test-suite/vsc
+
+    export RFM_CONFIG_FILE=$PWD/config_vsc.py
+    reframe --verbose --run \
+        --checkpath vsc.py \
+        --setvar VSCEnvTest.valid_systems='hortense:local' \
```

Comment:
I think the setvar options are no longer necessary if the valid systems are set in the tests themselves.

Reply:
yes, I agree, didn't want to touch launcher for other systems at the moment, experimenting on Vaughan